massa-ledger-worker/src/ledger_db.rs

```diff
@@ -244,6 +264,18 @@ impl LedgerDB {
     fn put_entry(&self, addr: &Address, ledger_entry: LedgerEntry, batch: &mut DBBatch) {
         let db = self.db.read();
 
+        // Version
+        //TODO: Get version number from parameters
```

Comment:
Maybe I don't really know it will be a futur concern I think

Reply:
Ok. I think it will be one of the things we have to think about before mainnet (a list of Components to version to make potential transitions after mainnet easier).